src/utils/serialize.ts

```diff
@@ -18,5 +18,5 @@ export const serialize = value => {
 
 /// Deserialize a function string and invoke the resulting object recreator.
 export const deserialize = value => {
-  return parse(value);
+  return value === 'null' || value === null ? null : parse(value);
```

Comment:
Curious if this was related to HMR or does it potentially resolve some of the reported regressions?

Reply:
@andrewthauer This just avoids passing the `"null"` string into `parse` which throws an error. This only occurs (the `value` of `"null"`) when some error has happened in the backend (I think  with the serialization).